app.py

```diff
@@ -413,7 +448,7 @@ def on_join(data):
     mymessage = {
         'username':data['username']
     }
--
+
     emit('join_room', json.dumps(mymessage), room=room)
```

Comment:
Maintain a list of all possible events as enumerations and use them here

Reply:
I am not understanding this?
All rooms are maintened independent and exclusive from each other.
I think you want to have a lost like what rooms are active.
But since rooms are active as soon as 1 person joins them so they are already in a maintened state. And to check events we can simply query database.
Here missing thing is to check if that event has expired, it will added.

If there’s something more than please tell
@L04DB4L4NC3R

Reply:
No, no. I am simply talking about enumerating socket events like `join_room`

Reply:
@L04DB4L4NC3R enumerating?
only such word i know is enumeration. please tell what you want to be done here. in human terminology

Reply:
I mean using `enums`. NVM, I am merging this, you can do that in future pull requests